packages/not-ocamlfind/not-ocamlfind.0.08/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "A small frontend for ocamlfind that adds a few useful commands"
+license: "MIT"
+maintainer: "Chet Murthy <chetsky@gmail.com>"
+
+(* Gerd wrote most of this code; I just modified it (and probably
+introduced bugs.  This is to silence opam *)
+
+authors: "Chet Murthy <chetsky@gmail.com>"
```

Comment:
Regarding the comment, the authors field can also be a list of authors so you could include Gerd as well in a future PR.